### PR TITLE
refactor: add og property and icon

### PR DESCRIPTION
**Pull request by SumitVerma-coder**
Added a favicon icon to the website.
image

### CODE REVIEW

**Comment by malivinayak** [comment]
Add  four basic open graph tags in this PR, then I will merge the PR

**Review by malivinayak — CHANGES_REQUESTED** [review]
Add four basic open graph property in order to fulfill the website meta info

**Comment by SumitVerma-coder** [comment]
Added four open graph tags

File: tic-tac-toe.png
[the remark this answers is not in the document]
Comment (malivinayak) [reply tic-tac-toe.png:1]: store it at assets/logo or assets/icon

**Review by malivinayak — CHANGES_REQUESTED** [review]

**Comment by Mouli-3542** [comment]
@malivinayak any new issues to work on

File: index.html [hunk 1]
```diff
@@ -4,8 +4,13 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta property="og:title" content="Tic Tac Toe" />
+    <meta property="og:type" content="website" />
+    <meta property="og:description" content="Play Tic Tac Toe Game online!" />
+    <meta property="og:image" content="https://store-images.s-microsoft.com/image/apps.2005.14057826194083709.67242c47-4fd7-4f1a-9dd6-5d93f6cc10df.f80f14c0-72ab-46ff-86cd-9d801c8e04e8?mode=scale&q=90&h=300&w=300" />
```
[the remark this answers is not in the document]
Comment (SumitVerma-coder) [reply index.html]: Moved all the images to assets/Images folder

[the remark this answers is not in the document]
Comment (SumitVerma-coder) [reply index.html]: image

**Review by malivinayak — APPROVED** [review]

**Pull request merged**